### ISSUE TITLE
Color Contrast Non-WCAG accessible in RPres Default Splash Screen

### ISSUE THREAD

**Issue by cgrudz** [rstudio/rstudio#5443]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.2.5001
    OS Version      :  Ubuntu 18.04.3 LTS
    R Version       :  3.4.4 (2018-03-15)

### Steps to reproduce the problem
Open the default .rpres template document and compile.

### Describe the problem in detail
The color settings for the splash/ title screen of the default .rpes template are not accessible to visually impaired users according to Web Content Accessibility Guidelines.  Specifically, the contrast level of the white text on light blue is too low. See e.g.,

https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=96A2B6

### Describe the behavior you expected
The default template should amend this to have higher contrast text in the opening slide.

**Comment by ronblum** [comment]
@cgrudz Thank you for raising this and the other accessibility issues. We are currently working on improvements to accessibility in RStudio—you'll note that @gtritchie tagged the ticket as an [accessibility issue](https://github.com/rstudio/rstudio/labels/accessibility) and an item for the project. So we appreciate your reporting any needs for improvement to accessibility that you come across!

**Comment by cgrudz** [comment]
No problem, we are working on better mathematical typesetting for accessibility at the University of Nevada, so I'll flag things as they come up in my workflow.

Cheers
